CONTRIBUTING.md

```diff
@@ -221,21 +221,43 @@ To update goldens for all tests, run:
 flutter test test/ --update-goldens
 ```
 
-## Manual Testing
+## Manual testing
 
 To explore DevTools with all experimental features enabled:
 
+### Setup and start
+
 1. [Configure](https://docs.flutter.dev/get-started/install) Dart or Flutter.
 
-2. Start DevTools:
+2. In your terminal `cd` to a folder where you want to clone devtools, and that does not have subfolder `devtools` yet. 
+
+3. Start DevTools:
+```
+git clone https://github.com/flutter/devtools.git;
```

Comment:
I do not think we should point users to use https. We should default to SSH and then have a section about HTTPS if they do not have an SSH key configured.

Reply:
This script target users like Mindy. I want to give them a fastest way to kick off devtools. They do not have SSH configured.